### PR TITLE
Fixes new requirements in requirements.txt, as well as 3 missing null checks, and the styles for the iam graphing

### PR DESCRIPTION
**Pull request by commjoen**
This is an extended fix of what we see in #879 : it fixes the 3 missing empty/null checks as well as the requirements.txt, as well as the styles reference for the iam Graphing

### CODE REVIEW

**Review by w0rmr1d3r — APPROVED** [review]
Nice! Take into account, that the changes done in the `requirements.txt` file, have already been requested in this PR https://github.com/duo-labs/cloudmapper/pull/876

**Comment by commjoen** [comment]
@w0rmr1d3r that is true, but given that this mr has not been merged for the last 3 weeks, I figured: for those that want a "1-stop-shop-departure": here you go :).

**Comment by w0rmr1d3r** [comment]
Great @commjoen !
Take a look, that there are a lot of PRs fixing some open issues, but haven't been reviewed :/ and not only for the last 3 weeks, I'm talking about months here.

**Comment by commjoen** [comment]
@w0rmr1d3r the only problem i might have to fix for now (which does not owrk for me) is visualizing the iAM graph.. was there an MR that already fixed this :) ?

**Comment by w0rmr1d3r** [comment]
Can't find any PR regarding that, neither any issue.
Have you opened one @commjoen ?

**Comment by commjoen** [comment]
Not yet, as i was trying to figure out what was going wrong... but still did not find an answer (as occupied by mulitple things here in a security team :P )

**Comment by commjoen** [comment]
See #1063114 : this fixes the styles so you can generate graphs for iam again.

**Comment by w0rmr1d3r** [comment]
Perfect @commjoen !

**Comment by commjoen** [comment]
@w0rmr1d3r do you know whether there is a way to visualize trust relationships between roles? because i am missing half of them.. but no errors anywhere...

**Comment by w0rmr1d3r** [comment]
Don't know at the moment 😕

**Comment by commjoen** [comment]
Any updates on this MR?

File: web/js/cloudmap.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ $(window).on('load', function(){
     
     $.when(
         $.getJSON("./data.json"),
-        $.getJSON("./style.json")
+        $.getJSON("./../style.json")
```
Comment (0xdabbad00) [inline web/js/cloudmap.js:40]: I don't think this is correct as it will break the other visualizations.

Comment (commjoen) [reply web/js/cloudmap.js:40]: The problem was that the script errorred that it could not find style.json, therefore i changed the reference.

Comment (commjoen) [reply web/js/cloudmap.js:40]: Does it work at your machine without the `...` ? or was there something else i should have done to make it work?

**Pull request closed without merging**